.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Run tests
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
```

Comment:
can we update this to use ubuntu 20 (for consistency across all edX repos) and set up a matrix with OS version and python version?